buildSrc/version.properties

```diff
@@ -1,5 +1,5 @@
 # When updating elasticsearch, please update 'rest' version in core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
```

Comment:
Just looking at this PR and stumbled across this comment, but it seems the file in question doesn't seem to need to get changed any more (looking at the logs this seems to not be necessary anymore since c9964d17), so maybe this comment can be removed as well?